packages/js/email-editor/package.json

```diff
@@ -0,0 +1,91 @@
+{
+	"name": "@mailpoet/email-editor",
+	"private": true,
+	"sideEffects": [
+		"build-style/**",
+		"src/**/*.scss"
+	],
+	"version": "0.1.0",
+	"scripts": {
+		"build:editor": "wp-scripts build && cp -Rf assets/ build",
+		"start": "wp-scripts start",
+		"lint:css": "wp-scripts lint-style",
+		"lint:css-fix": "wp-scripts lint-style --fix",
+		"lint:js": "wp-scripts lint-js './src/**/*.{js,jsx,json,ts,tsx,yml,yaml}'",
+		"lint:js-fix": "wp-scripts lint-js './src/**/*.{js,jsx,json,ts,tsx}' --fix",
+		"check-types": "NODE_OPTIONS=--max_old_space_size=2048 tsc --noEmit",
+		"format": "./node_modules/.bin/prettier --config .prettierrc.js --write 'src/**/*.{js,jsx,json,ts,tsx,yml,yaml,scss}'",
+		"build": "pnpm --if-present --workspace-concurrency=Infinity --stream --filter=\"$npm_package_name...\" '/^build:project:.*$/'",
+		"build:project": "pnpm --if-present '/^build:project:.*$/'",
+		"build:project:editor": "pnpm build:editor",
```

Comment:
We also using `wireit` (please reference to other packages) for wrapping regular project build steps as mater of optimizing build performance and need to align on using it in this package as well.

Reply:
This is noted. We planned on doing this in another PR.  More info here https://github.com/woocommerce/woocommerce/pull/55598#discussion_r1959896833